cedar-drt/Cargo.toml

```diff
@@ -10,10 +10,10 @@ highway = "0.8.1"
 env_logger = "0.10"
 log = "0.4"
 libfuzzer-sys = "0.4"
-cedar-policy = { path = "../cedar/cedar-policy", version = "2.0.0", features = ["integration_testing"] }
-cedar-policy-core = { path = "../cedar/cedar-policy-core", version = "2.0.0" }
-cedar-policy-validator = { path = "../cedar/cedar-policy-validator", version = "2.0.0" }
-cedar-policy-formatter = { path = "../cedar/cedar-policy-formatter", version = "2.0.0" }
+cedar-policy = { path = "../cedar/cedar-policy", version = "2.*", features = ["integration_testing"] }
+cedar-policy-core = { path = "../cedar/cedar-policy-core", version = "2.*" }
+cedar-policy-validator = { path = "../cedar/cedar-policy-validator", version = "2.*" }
+cedar-policy-formatter = { path = "../cedar/cedar-policy-formatter", version = "2.*" }
```

Comment:
Which version do we want to use here? I opted for `2.*` because it won't require us to update this repo with new minor versions.

Reply:
Seems reasonable to me.